article/helpers.py

```diff
@@ -8,25 +9,93 @@
 from . import structure
 
 
+WORDS_PER_MINUTE = 180  # Average WPM on screen
+
+
 def markdown_to_html(markdown_file_path):
     html = render_to_string(markdown_file_path)
     return markdown2.markdown(html)
 
 
+def filter_lines(lines_list):
+    """BeautifulSoup returns \n as lines as well, we filter them out.
+
+    It's a function because more filtering can be added later.
+    """
+    return filter(lambda x: x != '\n', lines_list)
+
+
+def lines_list_from_html(html):
+    """Parses the HTML to return text lines."""
+    soup = BeautifulSoup(html, 'html.parser')
+    return soup.findAll(text=True)
+
+
+def count_average_word_number_in_lines_list(lines_list, word_length=5):
+    """Assume average word length, counts how many words in all the lines."""
+    total_words = 0
+    for line in lines_list:
+        total_words += len(line)/word_length
+    return total_words
+
+
+def time_to_read_in_minutes(article):
+    """Return rounded time to read in minutes give an Article object."""
+    html = markdown_to_html(article.markdown_file_path)
+    lines_list = lines_list_from_html(html)
+    filtered_lines_list = filter_lines(lines_list)
+    total_words_count = count_average_word_number_in_lines_list(
+        filtered_lines_list
+    )
+    return round(total_words_count / WORDS_PER_MINUTE)
+
+
+def total_time_to_read_multiple_articles(articles):
+    return sum((time_to_read_in_minutes(article) for article in articles))
```

Comment:
`sum((article.time_to_read for article in articles))`

Reply:
Nice catch!